server/src/client_api.rs

```diff
@@ -35,7 +35,7 @@ pub async fn get_features(
         .ok_or(EdgeError::AuthorizationDenied)?;
     match req.app_data::<Data<FeatureRefresher>>() {
         Some(refresher) => refresher
-            .features_for_token(&validated_token)
+            .features_for_token(validated_token)
```

Comment:
would this compile later on, since we reuse the validated token further down in this method? I see since you're making a PR against another PR, our workflow isn't triggering.

Reply:
I run test locally and they passed, but I'm also not thaaat confident in my skills with Rust